git-gutter.el

```diff
@@ -951,13 +963,16 @@ start revision."
     (with-temp-file tmpfile
       (insert content))))
 
-(defsubst git-gutter:original-file-content (file)
```

Comment:
This is the crux of the changes. I'm not sure whether git updated and made this function incorrect, but the change creates the command `git show :0:<file>`, where `<file>` is relative to the root git directory. The file must be relative to the root directory for it to work correctly.

I just checked `man 7 gitrevisions`, and it looks like git actually defaults to `:0:`, which means to use the most recent revision, so this could be changed to just `(concat ":" (file-relative-name file repo-dir)`. However, I prefer to keep the `:0:` so it is more explicit.